charts/refarch-templates/templates/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+{{- $dot := . }}
+{{- range $module := .Values.modules -}}
+{{ $data := dict "dot" $dot "module" $module }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "getFullname" $data }}
+  labels:
+    {{- include "getLabels" $data | trim | nindent 4 }}
+spec:
+  {{- if not (and $module.autoscaling (or $module.autoscaling.targetCPUUtilizationPercentage $module.autoscaling.targetMemoryUtilizationPercentage))}}
+  replicas: {{ $module.replicas | default 1 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "getSelectorLabels" $data | trim | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "getLabels" $data | trim | nindent 8 }}
+    spec:
+      {{- if $module.initContainers }}
+      initContainers:
+        {{ toYaml $module.initContainers | nindent 8 }}
+      {{- end }}
+      {{- with $.Values.imagePullSecrets }}
+      imagePullSecrets:
```

Comment:
wenn etwas in Openshift importiert wird, dann gibt es eine Default-Reihenfolge . Vielleicht ist es sinnvoll die Reihenfolge auch im Helm Chart zu übernehmen

Reply:
sorry link wurde nicht reinkopiert https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting

Reply:
![grafik](https://github.com/user-attachments/assets/3dd7bb75-dbe5-4806-b92a-dbf5ad4d7759)

ja das man die Werte schneller findet.

Reply:
![grafik](https://github.com/user-attachments/assets/271608e2-11b9-47b9-878c-52574b70a192)

also es sind schon ein paar Unterschiede. Das Vergleichen fällt leichter